nmosnode/authclient.py

```diff
@@ -0,0 +1,193 @@
+# Copyright 2019 British Broadcasting Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import os
+import json
+import requests
+from requests.exceptions import HTTPError
+from six.moves.urllib.parse import urljoin
+from authlib.flask.client import OAuth
+
+from mdnsbridge.mdnsbridgeclient import IppmDNSBridge
+from nmoscommon.logger import Logger
+
+CREDENTIALS_PATH = os.path.join('/var/nmosauth/', 'oauth_credentials.json')
```

Comment:
Will this path exist? Is there already something in nmosnode which ensures this, and is there no danger of the path being used by other services which run as another user?

Reply:
This is gonna be `/var/nmosnode` with the directory being produced by the setup.py file eventually